tokens/movistar.json

```diff
@@ -573,14 +573,14 @@
       "description": "movistarBlue55"
     },
     "buttonSecondaryBackgroundHover": {
-      "value": "rgba({palette.movistarBlue}, 0.2)",
+      "value": "rgba({palette.white}, 0.15)",
```

Comment:
i don't understand this change, why do you use white in secondary backgroundHover?